.gitignore

```diff
@@ -13,6 +13,7 @@ __pycache__/
 .noseids
 .python-version
 .vscode/*
+.venv/*
```

Comment:
the entry for `venv` (no dot) is down on line 66, and it doesn't include the `/*` at the end. I wonder if that `/*` is necessary, and if so what it's doing? I'll have a look at the docs.

Reply:
According to [the docs](https://git-scm.com/docs/gitignore#_pattern_format) it seems like it's equivalent for our purposes. I guess we should just be consistent. Technically `.venv` would match a directory or file with that name. `.venv/` would match only a directory. `.venv/*` matches anything directly inside that directory up to another slash. `.venv/**` would match everything inside the dir no matter how deep.

Reply:
Updated to remove `/*`